sdk_contrib/koa/README.md

```diff
@@ -0,0 +1,59 @@
+
+## Requirements
+
+  AWS X-Ray SDK Core (aws-xray-sdk-core)
+  Koa 2.x or greater
+
+## AWS X-Ray and Koa
+
+The AWS X-Ray Koa package automatically records information for incoming and outgoing
+requests and responses, via the middleware functions in this package. To configure sampling, 
+dynamic naming, and more see the [set up section](https://github.com/aws/aws-xray-sdk-node/tree/master/packages/core#setup).
+
+The AWS X-Ray SDK Core has two modes - `manual` and `automatic`.
+Automatic mode uses the `cls-hooked` package and automatically
+tracks the current segment and subsegment. This is the default mode.
+Manual mode requires that you pass around the segment reference.
+
+In automatic mode, you can get the current segment/subsegment at any time:
+    var segment = AWSXRay.getSegment();
+
+In manual mode, you can get the base segment off of the context object:
+    var segment = ctx.segment;
+
+## Middleware Usage
+
+The Koa X-Ray SDK provides one middlewares: `xrayKoa.openSegment(<name>)`. 
+This middleware will wrap all of the defined routes that you'd like to trace. 
+In automatic mode, the `openSegment` middleware *must* be the last middleware added
+before defining routes, otherwise issues with the `cls-hooked`
+context may occur.
+
+## Automatic mode examples
```

Comment:
Can you change the sample code to use markdown js-formatted code blocks instead of indenting, like the [other example code](https://github.com/aws/aws-xray-sdk-node/blame/master/packages/core/README.md#L89)?

Reply:
@willarmiros I don't understand this request. The sample code uses the same formatting that the other examples do AFAIK and it appears as a proper code example in github.

Reply:
@willarmiros Done